lib/kitchen/verifier/pester.rb

```diff
@@ -129,6 +129,10 @@ def run_command_script
           $result = Invoke-Pester -OutputFile $OutputFilePath -OutputFormat NUnitXml -Path $TestPath -Passthru
           $result | Export-CliXml -Path (Join-Path -Path $TestPath -ChildPath 'result.xml')
           $host.SetShouldExit($result.FailedCount)
+          if( $result.FailedCount )
+          {
+            Write-Error -Message ('Pester finished with {0} failing tests.' -f $result.FailedCount) -ErrorAction Stop
```

Comment:
I would be inclined to suggest using `throw` rather than `Write-Error -ErrorAction Stop` 🙂 

```suggestion
            throw 'Pester finished with {0} failing tests.' -f $result.FailedCount
```